content/8.x/admin/installation-guide/Downloading & Installing.adoc

```diff
@@ -36,11 +44,14 @@ operating system. Common web server users are as follows:
 ** `nobody` (Linux/Apache)
 ** `IUSR_computerName` (Windows/IIS)
 [NOTE]
-The commands/steps taken to set permissions differ depending on
-your operating system. If you are experiencing issues with setting
-permissions on your SuiteCRM instance, visit our support forums.
+The commands/steps taken to set permissions differ depending on your operating system. If you are experiencing issues with setting permissions on your SuiteCRM instance, visit our support forums.
```

Comment:
For consistency with the rest of the Docs, perhaps use this construct instead:

```
{{% notice note %}}
...
{{% /notice %}}
```
Also in all the other places where `[NOTE]` was introduced in the new v8 Docs...